recipes/diligent-fx/all/conanfile.py

```diff
@@ -0,0 +1,117 @@
+import os
+from conans import ConanFile, tools, CMake
+from conans.errors import ConanInvalidConfiguration
+import glob
+import shutil
+
+class DiligentFxConan(ConanFile):
+    name = "diligent-fx"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/DiligentGraphics/DiligentFx/"
+    description = "DiligentFX is the Diligent Engine's high-level rendering framework."
+    license = ("Apache 2.0")
```

Comment:
```suggestion
    license = ("Apache-2.0")
```

Use the official spdx name